src/store/builder/index.ts

```diff
@@ -361,9 +354,10 @@ export function computeBlockUiData(block?: IBlock | null) {
   }
 }
 
-export function computeBlockVendorUiData(block?: IBlock | null) {
+export function computeBlockVendorUiMetadata(_block?: IBlock | null): IFloipUIMetadata {
```

Comment:
I renamed the function to match the new property name `vendor_metadata.floip.ui_metadata`

Reply:
Good naming is important. Thank you for these changes 👍

Reply:
Awesome, thanks 🙏